static-site/src/sections/group-members-page.tsx

```diff
@@ -0,0 +1,139 @@
+  const [ roles, setRoles ] = useState<string[]>([]);
+  const [ members, setMembers ] = useState<GroupMember[]>([]);
+
+  useEffect(() => {
+    async function getGroupMembership(groupName: string) {
+      const headers = { headers: {"Accept": "application/json"}};
+      let roles, members = [];
+      try {
+        const [ rolesResponse, membersResponse ] = await Promise.all([
+          fetch(uri`/groups/${groupName}/settings/roles`, headers),
+          fetch(uri`/groups/${groupName}/settings/members`, headers)
+        ]);
+        if (!rolesResponse.ok) {
+          throw new Error(`Fetching group roles failed: ${rolesResponse.status} ${rolesResponse.statusText}`)
+        }
+        if (!membersResponse.ok) {
+          throw new Error(`Fetching group members failed: ${membersResponse.status} ${membersResponse.statusText}`)
+        }
+        roles = await rolesResponse.json();
+        members = await membersResponse.json();
+      } catch (err) {
+        const errorMessage = (err as Error).message
+        if(!ignore) {
+          setErrorMessage({
+            title: "An error occurred when trying to fetch group membership data",
+            contents: errorMessage})
+        }
+      }
+      return {roles, members};
+    }
+
+    let ignore = false;
+    getGroupMembership(groupName).then(result => {
+      if (!ignore) {
+        setRoles(result.roles);
+        setMembers(result.members);
+      }
+    })
+    return () => {
+      ignore = true;
+    };
+  }, [groupName]);
+
+  return (
+    <GenericPage banner={errorMessage.title ? <ErrorBanner {...errorMessage} /> : undefined}>
+      <FlexGridRight>
+        <splashStyles.Button to={uri`/groups/${groupName}`}>
+          Return to {`"${groupName}"`} Page
+        </splashStyles.Button>
+      </FlexGridRight>
+      <MediumSpacer/>
+
+      <splashStyles.H2>
+        {`"${groupName}"`} Group Members
+      </splashStyles.H2>
+      <BigSpacer/>
+
+      {roles && members
+        ? <MembersTable roles={roles as string[]} members={members as GroupMember[]} />
+        : <splashStyles.H4>Fetching group members...</splashStyles.H4>}
+    </GenericPage>
+  )
+};
+
+const MembersTable = ({ roles, members }: { roles: string[], members: GroupMember[]}) => {
+  const sortedMembers = members.toSorted((a, b) => a.username.localeCompare(b.username));
+  function mostPrivilegedRole(memberRoles: string[]) {
+    // Assumes that the provided roles are listed in order of least to most privileged
+    return memberRoles.reduce((a, b) => roles.indexOf(a) > roles.indexOf(b) ? a : b);
+  }
+
+  return (
+    <CenteredContainer>
+      <div className="row">
+        <div className="col-6">
+          <splashStyles.CenteredFocusParagraph>
+            <strong>Username</strong>
+          </splashStyles.CenteredFocusParagraph>
+        </div>
+        <div className="col-6">
+          <splashStyles.CenteredFocusParagraph>
+            <strong>Roles</strong>
```

Comment:
Some related suggestions:

1. Make the header singular. Only the most privileged role is shown per person.
2. Make the values singular. This reads better: "trs is an owner"
3. Capitalize the values. This looks more polished.

![image](https://github.com/user-attachments/assets/18641428-4b99-4da5-809c-849f85157f6e)

Reply:
All good suggestions! Updated in https://github.com/nextstrain/nextstrain.org/pull/973/commits/a2c9dc4094f9c4c0823407073b9615f93cc846de